config/routes.rb

```diff
@@ -75,7 +75,10 @@
     resources :fundraisers, only: :update
     resources :surveys, only: :update
     resources :texts, only: :update
-    resources :call_tools, only: :update
+    resources :call_tools, only: :update do
+      delete :delete_sound_clip, on: :member
```

Comment:
Having a `DELETE /call_tools/:id/delete_sound_clip` is redundant. The `DELETE` method is more than enough to indicate that you're deleting a resource: `DELETE /call_tools/:id/sound_clip`